sale_stock_on_hand_popup/views/sale_order_views.xml

```diff
@@ -8,7 +27,7 @@
         <xpath expr="." position="inside">
             <script
                 type="text/javascript"
-                src="/sale_stock_on_hand_popup/static/src/js/qty_at_date_widget.js"
+                src="/sale_stock_on_hand_popup/static/src/js/stock_on_hand_widget.js"
             />
         </xpath>
     </template>
```

Comment:
**quibble(non-blocking)**: This declaration is usually put in its own file.

Reply:
Moved to [sale_stock_on_hand_popup/views/assets.xml](https://github.com/OCA/stock-logistics-warehouse/pull/1929/files#diff-9d5e7a9589d7fb833fea11a65ad227cb34ed5414aad4e0caab73f7607b32e1ed) 

Let me know if it looks right to you